iothub/device/src/Exceptions/ExceptionHandlingHelper.cs

```diff
@@ -33,5 +33,10 @@ public static Task<string> GetExceptionMessageAsync(HttpResponseMessage response
         {
             return response.Content.ReadAsStringAsync();
         }
+
+        private static string CreateMessageWhenDeviceNotFound(string deviceId)
```

Comment:
this is to replace what a particular constructor [did ](https://github.com/Azure/azure-iot-sdk-csharp/blob/previews/v2/iothub/device/src/Exceptions/DeviceNotFoundException.cs#L58)in the `DeviceNotFoundException` class